### PR TITLE
[Search] Illustrate how to hide locations from catalog search

### PR DESCRIPTION
**Pull request by iamEAP**
## Hey, I just made a Pull Request!

This has come up a couple of times in discord!  May as well document it.

Follow-up to #6955

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by freben — APPROVED** [review]

File: docs/features/search/how-to-guides.md [hunk 1]
```diff
@@ -97,3 +97,37 @@ of the `SearchType` component.
...
+systems, groups, users, and other aspects of your software ecosystem. However,
+you may not always want _every_ aspect to appear when a user searches the
+catalog. Examples include:
+
+- Entities of kind `Location`, which are often not useful to Backstage users.
+- Entities of kind `User` or `Group`, if you'd prefer that users and groups be
+  exposed to search in a different way (or not at all).
+
+It's possible to write your own [Collator](./concepts.md#collators) to control
+exactly what's available to search, (or a [Decorator](./concepts.md#decorators)
```
Comment (emmaindal) [inline docs/features/search/how-to-guides.md:113]: Do you think its valuable to give an example of how to implement a new collator/decorator here? Like another how to guide. Could be done in a separate PR though!

**Review by emmaindal — APPROVED** [review]
Nice! ⭐

File: docs/features/search/how-to-guides.md (hunk 1 above)
Comment (iamEAP) [reply docs/features/search/how-to-guides.md:113]: For sure!  But maybe after #7239..... 😬

**Comment by bodilb** [comment]
Excellent! 👌

**Pull request merged**